src/igvfd/audit/file_set.py

```diff
@@ -176,3 +187,62 @@ def audit_inconsistent_seqspec(value, system):
                     f'but belong to different sequencing sets.'
                 )
                 yield AuditFailure('inconsistent sequence specifications', f'{detail} {description}', level='ERROR')
+
+
+@audit_checker('ConstructLibrarySet', frame='object')
+@audit_checker('PredictionSet', frame='object')
+def audit_loci_valid_chrom_sizes(value, system):
+    '''
+    [
+        {
+            "audit_description": "Loci are expected to belong to the same assembly.",
+            "audit_category": "inconsistent loci",
+            "audit_level": "ERROR"
+        },
+        {
+            "audit_description": "Loci are expected to specify a valid chromosome name and range from its respective assembly.",
+            "audit_category": "inconsistent loci",
+            "audit_level": "ERROR"
+        }
+    ]
+    '''
+    description_inconsistent_assembly = get_audit_description(audit_loci_valid_chrom_sizes, index=0)
+    description_inconsistent_loci = get_audit_description(audit_loci_valid_chrom_sizes, index=1)
+    GRCh38_chrom_sizes = load_chrom_sizes_file('src/igvfd/audit/_static/GRCh38.chrom.sizes')
+    GRCm39_chrom_sizes = load_chrom_sizes_file('src/igvfd/audit/_static/GRCm39.chrom.sizes')
+    invalid_chroms = []
```

Comment:
I think you can make it more elegant, define dictionary {'GRCh38':...,'RGRCm39':...} then instead of multiple if statements use the assembly as key of finding relevant chrom sizes